.circleci/run_tests.sh

```diff
@@ -29,6 +29,9 @@ PGPASSWORD=sharding_user pgbench -h 127.0.0.1 -U sharding_user shard2 -i
 LOG_LEVEL=error toxiproxy-server &
 sleep 1
 
+# Remove toxiproxy if it exists.
+toxiproxy-cli delete postgres_replica || true
```

Comment:
It is needed for local development, if you're going to run the tests multiple times.